src/style.css

```diff
@@ -1,3 +1,124 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  /* background-color: darkgray; */
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 202px;
+  height: 410px;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
```

Comment:
Consider using this font family for the entire document, to avoid code repeating in the future